core/schema/src/main/resources/org/hawkular/schema/updates/schema-0.27.0.groovy

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright 2014-2017 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+schemaChange {
+  version '8.0'
+  author 'burmanm'
+  tags '0.27.x'
+  cql """
+CREATE TABLE data_temp_0 (
```

Comment:
I would make the tags column ```frozen```.  It will result in the map being serialized into a single blob column vs being stored in multiple columns. It is more efficient both in terms of writes and in terms of space. The current data table does **not** use ```frozen``` which is an oversight on my part. The caveat with a ```frozen``` collection is that you cannot add/append/remove a single element. You have to update the entire collection. For data point tags I think this is perfectly fine.